src/gui/src/components/explorer-grid/overview-sidebar/suggested-queries.tsx

```diff
@@ -0,0 +1,130 @@
+import { useEffect, useState } from 'react'
+import { ChevronRight, Search, CornerDownRight } from 'lucide-react'
+import { Button } from '@/components/ui/button.tsx'
+import { Query } from '@vltpkg/query'
+import { QueryToken } from '@/components/query-bar/query-token.tsx'
+import { useGraphStore } from '@/state/index.ts'
+import { AnimatePresence, motion } from 'framer-motion'
+
+interface SuggestedQuery {
+  name: string
+  description: string
+  query: string
+}
+
+export const queries: SuggestedQuery[] = [
+  {
+    name: 'Outdated',
+    description: 'Find packages that are outdated',
+    query: ':outdated(major)',
+  },
+  {
+    name: 'Unmaintained',
+    query: ':unmaintained',
+    description: 'Find packages that are unmaintained',
+  },
+  {
+    name: 'System access',
+    query: ':has(:fs) :has(:env)',
```

Comment:
I think the space here is a mistake?

```suggestion
    query: ':has(:fs):has(:env)',
```

Reply:
Agreed that the space seems like it's a mistake... that said, we should actually be able to can nest an `:is()` inside of `:has()` to remove the multiple `:has` statements (ie. `:has(:is(:fs,:env))`) that said, you should likely be looking up elements themselves & not elements with children that match this criteria.

ie. `:is(:fs, :env)` is likely the more appropriate selector which will match parents & children that have file system or environment variable access.